GenderPayGap.WebUI/Controllers/Admin/AdminOrganisationReturnController.cs

```diff
@@ -111,5 +118,60 @@ private static string SanitiseOrganisationNameForFilename(string organisationNam
             return organisationName;
         }
 
+        [HttpGet("organisation/{id}/returns/{year}/delete")]
+        public IActionResult DeleteReturnsOfAYearGet(long id, int year)
+        {
+            var organisation = dataRepository.Get<Organisation>(id);
+            List<long> returnsId = organisation.Returns.Where(r => r.AccountingDate.Year == year).Select(r => r.ReturnId).ToList();
+
+            var viewModel = new AdminDeleteReturnViewModel {Organisation = organisation, ReturnsId = returnsId, Year = year};
+
+            return View("DeleteReturns", viewModel);
+        }
+
+        [HttpGet("organisation/{id}/returns/{year}/delete/{returnId}")]
+        public IActionResult DeleteReturnGet(long id, int year, long returnId)
+        {
+            var organisation = dataRepository.Get<Organisation>(id);
+            List<long> returnsId = organisation.Returns.Where(r => r.AccountingDate.Year == year && r.ReturnId == returnId)

```

Comment:
This may seem a bit odd but the reason I did it was for validation

Reply:
That's a great idea.
I've changed this code so that this purpose (validation) is clearer.